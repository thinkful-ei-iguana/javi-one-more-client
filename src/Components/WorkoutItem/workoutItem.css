*{
    margin: 0;
    padding: 0;
}

body{
    box-sizing: border-box;
}

li{
    margin-top: 10px;
    list-style: none;
}