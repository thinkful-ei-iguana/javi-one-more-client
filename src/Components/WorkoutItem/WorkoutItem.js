import React from 'react'
import '../WorkoutItem/workoutItem.css'
import { Link } from 'react-router-dom'

export default function WorkoutItem(props){
    
        return (
            <div>
                <li >
                 <Link to={`/workout/${props.id}`}> <h3>{props.title}</h3> </Link>
                </li>
                <span><Link to={`/edit/${props.id}`}>Edit</Link></span>
            </div>
        )
    }

