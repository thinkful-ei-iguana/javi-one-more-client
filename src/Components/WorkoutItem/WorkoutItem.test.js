import React from 'react';
import ReactDOM from 'react-dom';
import { BrowserRouter } from 'react-router-dom'
import WorkoutItem from '../WorkoutItem/WorkoutItem';
import renderer from 'react-test-renderer'


describe('GET /workouts' , () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<BrowserRouter><WorkoutItem /></BrowserRouter>, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<BrowserRouter><WorkoutItem /></BrowserRouter>)
      .toJSON()
    expect(tree).toMatchSnapshot();
  })
})