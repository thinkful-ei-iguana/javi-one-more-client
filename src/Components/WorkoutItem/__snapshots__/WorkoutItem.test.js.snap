// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GET /workouts renders the UI as expected 1`] = `
<div>
  <div>
    <a
      href="/workout/undefined"
      onClick={[Function]}
    >
       
      <h3
        className="workout-title"
      />
    </a>
    <a
      href="/edit/undefined"
      onClick={[Function]}
    >
      <span
        className="edit-button"
      >
        Edit
      </span>
    </a>
  </div>
</div>
`;
